resources/definitions/dagoma_discoeasy200.def.json

```diff
@@ -30,24 +30,23 @@
         },
         "machine_head_with_fans_polygon": {
             "default_value": [
-                [16, 37],
-                [16, -65],
-                [-16, -65],
-                [16, 37]
+                [17, 70],
+                [17, -40],
+                [-17, -40],
+                [17, 70]
             ]
         },
         "gantry_height": {
-            "default_value": 55
-        },
-        "machine_gcode_flavor": {
-            "default_value": "RepRap"
+            "default_value": 10
         },
         "machine_start_gcode": {
-            "default_value": ";Gcode by Cura\nG90   ;absolute positioning\nM106 S250 ;fan on for the palpeur\nG28 X Y\nG1 X50\nM109 S180\nG28\nM104 S{print_temperature}\n;Activation palpeur\n;bloc palpeur\nG29 ;Auto level\nM107 ;start with the fan off\nG1 X100 Y20 F3000\nG1 Z0.5\nM109 S{print_temperature}\nM140 S{material_bed_temperature}\nM82 ;set extruder to absolute mode\nG92 E0 ;zero the extruded length\nG1 F200 E10 ;extrude 10mm of feed stock\nG92 E0 ;zero the extruded length again\nG1 Z3\nG1 F3000\n"
```

Comment:
The `material_bed_temperature` is gone. Is it meant to be so?

Reply:
Hi LipuFei,

Yes as the discoeasy200 has no heating bed.

Regards,
Orel